package.json

```diff
@@ -32,7 +35,10 @@
     "react-native-reanimated": "^3.3.0",
     "react-native-safe-area-context": "^4.6.3",
     "react-native-screens": "^3.22.0",
-    "react-native-vector-icons": "^10.0.0"
+    "react-native-svg": "12.1.1",
+    "react-native-vector-icons": "^10.0.0",
+    "react-redux": "^9.1.2",
```

Comment:
I am concerned about using `redux`.

The primary reason being the up skilling needed across the company to use redux effectively.

Reply:
Which tooling you think which should go for @jjalan, any suggestions?

Reply:
In web, we are using React native context's.

Does that work in React Native?